tools/run-mypy

```diff
@@ -94,6 +95,23 @@ PY3_VENV_DIR = "/srv/zulip-py3-venv"
 MYPY_VENV_PATH = os.path.join(PY3_VENV_DIR, "bin", "mypy")
 if six.PY2 and os.path.exists(MYPY_VENV_PATH):
     mypy_command = MYPY_VENV_PATH
+
+    # If we are using zulip-py3-venv we will force the use of the correct version
+    with open(os.path.join(TOOLS_DIR, "..", "requirements", "mypy.txt")) as mypy_req:
+        for line in mypy_req:
+            if line.startswith("git+https://github.com/python/mypy.git"):
+                required_mypy_version = line.rsplit("@")[-1].strip()
```

Comment:
I guess the other approach would be to use the entire requirements.txt line as the "version", and just use a stamp file to keep track of which one is currently installed.  We pretty frequently install something from git without a real version number to work around mypy bugs...